gamemodes/PPC_Trucking.pwn

```diff
@@ -34,6 +34,8 @@ new PlayersBeforePolice	= 0;
 #include <sscanf2>
 #include <streamer>
 
+native IsValidVehicle(vehicleid);
```

Comment:
There is no need to use `IsValidVehicle` as the gamemode has an own function to check if a vehicle exists.